src/components/ui/project/irysmanga/IrysMangaDataWrapper.tsx

```diff
@@ -37,7 +37,7 @@ async function fetchOptimizedImageURLs({ project }: IProps) {
 				chapter.id,
 				chapter.pages.map((page) => {
 					if (bypassImgProxy) {
-						return page;
+						return page.replace(/en\/0([4-9]\.png)/, 'en/$1');
```

Comment:
Don't think this will affect anything, needed this to fix some devprops issues I was having.